### DISCUSSION TITLE
B站测试抢票提示系统找不到指定的文件

### DISCUSSION THREAD

**Discussion by bda3456** [mikumifa/biliTickerBuy#438] [提问专区]
是我电脑少下载了什么配置吗？
<img width="952" alt="微信图片_20240930000815" src="https://github.com/user-attachments/assets/4a7a57df-e7c7-492e-a31f-6e22db9e0241">

**Comment by mikumifa** [comment]
播放音乐问题，因为b站不能退票，没有回流，所以这个功能修复放一放，不急

**Reply by bda3456** [reply]
好滴，谢谢大佬
